src/collections/members/ashish-tiwari/index.mdx

```diff
@@ -0,0 +1,17 @@
+---
+name: Ashish Tiwari
+position: Contributor, Community
+image_path: ./ashish-tiwari.jpg
+github: revolyssup
+linkedin: at1511
+twitter: Ashisht31418366
+location: New Delhi, India
+bio: I am a final year computer science undergraduate at IP University, Delhi. Passionate Backend Developer with an interest in microservices and cloud-native ecosystems. Constantly learning new things based on interest & curiosity and trying to give back as much in open-source. Other than that, I love solving algorithm problems. A side note about me would be that, once upon a time, I fell in love with mathematics and physics, which still is an ongoing affair.
+badges:
+ - contributor
+ - community
```

Comment:
Hey @Rubix982 I think we need to remove this section for now because right now we do not have any contributor badge.

Reply:
Noted!

My bad, I mixed up the `position` and `badges`. 

Thanks for the review!

Reply:
Made changes as suggested by @leecalcote